x-pack/plugins/lens/public/visualizations/xy/xy_config_panel/axis_settings_popover.test.tsx

```diff
@@ -67,13 +67,23 @@ describe('Axes Settings', () => {
   });
 
   it('has the tickLabels switch on by default', () => {
-    const component = shallow(<AxisSettingsPopover {...props} />);
+    const component = shallow(
+      <AxisTicksSettings
+        axis={props.axis}
+        isAxisLabelVisible={props.areTickLabelsVisible}
+        updateTicksVisibilityState={jest.fn()}
+      />
+    );
     expect(component.find('[data-test-subj="lnsshowxAxisTickLabels"]').prop('checked')).toBe(true);
   });
 
   it('has the tickLabels switch off when tickLabelsVisibilitySettings for this axes are false', () => {
     const component = shallow(
-      <AxisSettingsPopover {...props} axis="yLeft" areTickLabelsVisible={false} />
+      <AxisTicksSettings
+        axis="yLeft"
```

Comment:
Once the comment above changed, this can be reverted as well.

Reply:
Same